rpgJavaInterpreter-core/src/test/kotlin/com/smeup/rpgparser/interpreter/RecordTest.kt

```diff
@@ -0,0 +1,36 @@
+package com.smeup.rpgparser.interpreter
+
+import kotlin.test.Test
+import kotlin.test.assertFalse
+import kotlin.test.assertTrue
+
+class RecordTest {
+
+    @Test
+    fun matchesOK() {
+        val record = Record(
+            Field("F1", StringValue("A")),
```

Comment:
I wonder if you could use a Pair instead of Field

Reply:
Originally I wrote the code using Pair, but then the it seemed a bit obscure to me: I think that 
x.name, x.value are more understandable than x.first, x.second, see for example the matches fun